### PR TITLE
Flow to run specint

### PR DESCRIPTION
**Pull request by vb000**

### CODE REVIEW

File: regression/specint/rules.mk [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# This Makefile fragment defines rules for building RISC-V binaries
+# associated with the tests in this sub-directory
+
+# SPMD_SRC_PATH is the path to the software/spmd directory in BSG
+# Manycore. It contains directories with RISC-V programs.
+SPMD_SRC_PATH = $(BSG_MANYCORE_DIR)/software/spmd/specint2000/
```
Comment (drichmond) [inline regression/specint/rules.mk]: Can you change this to SPECINT_SRC_PATH?

File: regression/specint/rules.mk [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+$(USER_CLEAN_RULES): 
+	CL_DIR=$(CL_DIR) \
+	BSG_MANYCORE_DIR=$(BSG_MANYCORE_DIR) \
+	BASEJUMP_STL_DIR=$(BASEJUMP_STL_DIR) \
+	IGNORE_CADENV=1 \
+	BSG_MACHINE_PATH=$(BSG_MACHINE_PATH) \
+	$(MAKE) -C $(SPMD_SRC_PATH) -f Makefile.$(subst .clean,,$(subst test_,,$@)) clean
+
+$(SPMD_SRC_PATH)/%.riscv: $(CL_DIR)/Makefile.machine.include
```
Comment (drichmond) [inline regression/specint/rules.mk]: Ditto here

File: regression/specint/spmd_tests.h [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#ifndef __SPMD_TESTS_H
```
Comment (drichmond) [inline regression/specint/spmd_tests.h]: I think this file should be called specint_tests.h

File: testbenches/specint/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Specint
```
Comment (drichmond) [inline testbenches/specint/README.md]: I updated the README's in a recent PR. Can you match the format in those?

**Comment by vb000** [comment]
@drichmond pushed suggested edits.

**Pull request merged**